### PR TITLE
Update acro, manual, rattitude modes

### PR DESCRIPTION
**Pull request by hamishwillee**
Last major set of modes to be updated. Adds remaining docs for individual flight modes.

### CODE REVIEW

File: en/flight_modes/README.md [hunk 1]
```diff
@@ -409,76 +422,3 @@ Abbreviations:
 
 
 
-<!--
```
Comment (hamishwillee) [inline en/flight_modes/README.md]: FYI, this was previously commented out. Just removing it now information has been integrated.

File: en/flight_modes/acro_fw.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+
+<!--
```
Comment (hamishwillee) [inline en/flight_modes/acro_fw.md]: Below are possibly relevant parameters. Should these be made visible? Are there any others?

File: en/flight_modes/acro_mc.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+RC RPY stick inputs control the rate of angular rotation around the respective axes. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (not necessarily level!)
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+ 
+<!--
```
Comment (hamishwillee) [inline en/flight_modes/acro_mc.md]: Below are possibly relevant parameters. Should these be made visible? Are there any others?

File: en/flight_modes/manual_fw.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+This is the only mode that overrides the FMU (commands are sent via the safety coprocessor). It provides a safety mechanism that allows full control of throttle, elevator, ailerons and rudder via RC in the event of an FMU firmware malfunction.
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+<!-- Possibly of interest:
```
Comment (hamishwillee) [inline en/flight_modes/manual_fw.md]: Below are possibly relevant parameters. Should these be made visible? Are there any others?

File: en/flight_modes/rattitude_mc.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* Sticks outside threshold (like *Acro mode*): 
+  * RPY stick inputs control the rate of angular rotation around the respective axes.
+
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
```
Comment (hamishwillee) [inline en/flight_modes/rattitude_mc.md]: Below are possibly relevant parameters. Should these be made visible? Are there any others? 

Specifically, do all the params for stabilised and for acro also apply here?

**Comment by tubeme** [comment]
Despite now the difficulty level is much better I think in reality there are only three levels of difficulty. This BTW indicator concerns mostly the novice or the experienced new users of PX4-QGC. So I think better not provide 6 scale for dificulty.

So it could be:

Easy - GREEN - Position, Altitude, Hold, Autonomous.
Medium - Yellow - FW Manual/Stabilized; MC Manual
Hard - Red - FW Acro, FW Manual; MC Acro, Rattitude

Plain and simple without numbers.

**Comment by tubeme** [comment]
It could be like this:

difficult level

**Comment by hamishwillee** [comment]
@bkueng Do you have an opinion on https://github.com/PX4/px4_user_guide/pull/191#issuecomment-377678813 ?

I don't feel strongly about this, except that I probably wouldn't have the icon at all on autonomous modes.

**Comment by bkueng** [comment]
I'm ok with 3 levels as well. Though the labels H, M, E are not obvious to me.
I'll review more detailed in the next few days.

**Comment by tubeme** [comment]
@bkueng The letters are Hard, Medium, Easy combined with a legend and color coding they are pretty much self explanatory. Using the ladder principle everybody knows from the traffic lights what is green yellow and red. @hamishwillee. Or we can omit the letters as well just leave the color square or we can use only circles. like this.
variant 2

**Comment by hamishwillee** [comment]
Thanks @tubeme . You've talked me into it :-) Updated as suggested using green, yellow, red icons. @bkueng Looking forward to your review when you have a chance.

File: en/flight_modes/acro_fw.md (hunk 2 above)
Comment (bkueng) [reply en/flight_modes/acro_fw.md]: This is a good set. In acro it's particularly important to know your maximum rates and stick curve (only linear for fw at the moment, we might add expo in future, as we have for mc)

File: en/flight_modes/acro_mc.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+![MC Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_MC.png)
+
+<!-- image above incorrect: https://github.com/PX4/px4_user_guide/issues/182 -->
+
+## Technical Description
+
+RC/manual mode for performing acrobatic maneuvers e.g. flips, rolls and loops.
+
+RC RPY stick inputs control the rate of angular rotation around the respective axes. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (not necessarily level!)
```
Comment (bkueng) [inline en/flight_modes/acro_mc.md]: missing full stop at the end

File: en/flight_modes/acro_mc.md (hunk 3 above)
Comment (bkueng) [reply en/flight_modes/acro_mc.md]: yes, all of these are important

File: en/flight_modes/manual_fw.md (hunk 4 above)
Comment (bkueng) [reply en/flight_modes/manual_fw.md]: I think FW_MAN_P_MAX and FW_MAN_R_MAX don't apply for fully manual mode. The others are relevant, but I'm not sure if there are more.

File: en/flight_modes/rattitude_mc.md (hunk 5 above)
Comment (bkueng) [reply en/flight_modes/rattitude_mc.md]: MC_RATT_TH should be mentioned here. All the others for manual and acro apply as well, but I think a link to these modes is sufficient.

**Comment by tubeme** [comment]
Here is one I made.
altimeter icon small

File: en/flight_modes/acro_fw.md (hunk 2 above)
Comment (hamishwillee) [reply en/flight_modes/acro_fw.md]: Thanks. Will add!

File: en/flight_modes/acro_mc.md (hunk 6 above)
Comment (hamishwillee) [reply en/flight_modes/acro_mc.md]: Fixed

File: en/flight_modes/acro_mc.md (hunk 3 above)
Comment (hamishwillee) [reply en/flight_modes/acro_mc.md]: Thanks. Added

**Comment by tubeme** [comment]
Here is another version which I think will scale OK. 
altimeter icon v2 small

**Comment by tubeme** [comment]
Extra small.
altimeter icon v2 extra small

**Comment by hamishwillee** [comment]
@tubeme Thanks, but they still don't work at the sizes used. The sizes to display are square 20px (small - group page) and 30px (sightly larger - individual altitude pages). You can see how they render [here](https://hamishwillee.gitbooks.io/ham_px4userguide/content/v/6fbec32592d3c73eaef2dbdc6764bdd9bdfafbc9/en/flight_modes/#altitude_fw)

Also reproduced below:
image

To match the other icons they need to be black/white, and have thicker white areas relative to black. Can you try again?

**Comment by hamishwillee** [comment]
@tubeme @bkueng I played around with the altitude icon based on the essential elements of your icon (e.g. "ALT" text, and a dial). This scales better - are you OK with it?

image

image

**Comment by bkueng** [comment]
Looks good to me.

**Pull request merged**